src/Orleans.Runtime/ReminderService/ReminderRegistry.cs

```diff
@@ -1,21 +1,36 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+
+using Microsoft.Extensions.DependencyInjection;
 using Orleans.Runtime.Services;
 using Orleans.Timers;
 
 namespace Orleans.Runtime.ReminderService
 {
     internal class ReminderRegistry : GrainServiceClient<IReminderService>, IReminderRegistry
     {
+        private const string ReminderServiceNotConfigured =
+           "The reminder service has not been configured. Reminders can be configured using extension methods from the following packages:"
```

Comment:
Not convinced we should be listing all of these in code.
No good way to keep this list accurate.
A reference to docs should be sufficient.